feat: break down description in problem & solution section (#6)
dialoguemd/.github: PULL_REQUEST_TEMPLATE.md

> ## Problem

Author: We are always solving problems:

## Problem

* As a member with access to the challenges, I want to see the list of challenges I can join and the oncoming ones.

## Solution

* Add a `/v1/challenges` endpoint which list ongoing and upcoming challenges by default.
* It does a single query on Dynamodb ...

<!-- What is the problem this PR solves?
     Intent: Readers outside your team can understand without opening JIRA or Notion.

>              It fastens reviewing and future troubleshooting. -->

Reviewer: ```suggestion
             It accelerates code review and future troubleshooting. -->
```

## Solution

> <!-- How is this PR solves the problem? What are the rationales behind that solution?

Reviewer 2: ```suggestion
<!-- How does this PR solve the problem? What is the rationale behind the solution?
```
